common/tier4_localization_rviz_plugin/src/pose_history/pose_history_display.hpp

```diff
@@ -27,20 +27,19 @@ namespace rviz_rendering
 {
 class BillboardLine;
 }  // namespace rviz_rendering
-namespace rviz_common
-{
-namespace properties
+namespace rviz_common::properties
 {
 class ColorProperty;
 class FloatProperty;
 class IntProperty;
 class BoolProperty;
-}  // namespace properties
-}  // namespace rviz_common
+}  // namespace rviz_common::properties
 
 namespace rviz_plugins
 {
-class PoseHistory : public rviz_common::MessageFilterDisplay<geometry_msgs::msg::PoseStamped>
+class PoseHistory
+: public rviz_common::MessageFilterDisplay<
+    geometry_msgs::msg::PoseStamped>  // NOLINT [cppcoreguidelines-special-member-functions]
```

Comment:
Defining a copy/move constructor/assign-operator can be used instead of NOLINT.
```cpp
PoseHistory(const PoseHistory &) = delete;
PoseHistory(const PoseHistory &&) = delete;
PoseHistory& operator=(const PoseHistory &) = delete;
PoseHistory& operator=(const PoseHistory &&) = delete;
```

Reply:
I  added.